art/defences/adversarial_trainer.py

```diff
@@ -26,6 +26,8 @@
               unbounded attacks (e.g., DeepFool), this can result in invalid (very noisy) samples being included.
 
 | Paper link: https://arxiv.org/abs/1705.07204
+
+| Please keep in mind the limitations of defences. Adversarial training is still an open research question.
```

Comment:
I would suggest to make the comment a little more specific, e.g.
```
Please keep in mind the limitations of defences. While adversarial training is widely regarded as a promising, principled approach to making classifiers more robust (see https://arxiv.org/abs/1802.00420), very careful evaluations are required to assess its effectiveness case by case (see https://arxiv.org/abs/1902.06705).
```

Reply:
This is a really good suggestion, I agree. I actually decided to add the evaluation paper to each defense comment, as it is a really good pointer here. Thank you!